src/languages/swift.js

```diff
@@ -224,24 +242,46 @@ export default function(hljs) {
     USER_DEFINED_ATTRIBUTE
   ];
 
+  // https://docs.swift.org/swift-book/ReferenceManual/Types.html
   const TYPE = {
-    className: 'type',
-    begin: '\\b[A-Z][\\w\u00C0-\u02B8\']*',
-    relevance: 0
-  };
-  // slightly more special to swift
-  const OPTIONAL_USING_TYPE = {
-    className: 'type',
-    begin: '\\b[A-Z][\\w\u00C0-\u02B8\']*[!?]',
-    relevance: 0
+    begin: lookahead(/\b[A-Z]/),
+    relevance: 0,
+    contains: [
+      { // Common Apple frameworks, for relevance boost
+        className: 'type',
+        begin: concat(/(AV|CA|CF|CG|CI|CL|CM|CN|CT|MK|MP|MTK|MTL|NS|SCN|SK|UI|WK|XC)/, Swift.identifierCharacter, '+')
```

Comment:
Nice!

Reply:
Add `\b` to make sure this matches only the start?

Reply:
Would that make a difference?
Is this not already handled by the `begin: lookahead(/\b[A-Z]/)` rule?

Reply:
I guess in this case it may not actually matter... just originally you say it should start with \b but then your inner rule is much wider... which given different child rules could become be a problem... but given the children we actually have `\b` might actually not change anything since I think the other things you allow are all word boundaries to being with.  So I think I flagged it as more of a future thing, but that might be inconsistent with how I've flagged other things, LOL.  We can just resolve this and leave it unchanged.